pcdsdevices/pseudopos.py

```diff
@@ -167,4 +168,23 @@ class SimDelayStage(DelayBase):
 
 class PseudoSingleInterface(PseudoSingle, FltMvInterface):
     """PseudoSingle with FltMvInterface mixed in."""
-    pass
+    notepad_setpoint = Cpt(
+        NotepadLinkedSignal, ':OphydSetpoint',
+        notepad_metadata={'record': 'ao', 'default_value': 0.0},
+    )
+
+    notepad_readback = Cpt(
+        NotepadLinkedSignal, ':OphydReadback',
+        notepad_metadata={'record': 'ai', 'default_value': 0.0},
+    )
+
+    def __init__(self, prefix='', parent=None, **kwargs):
+        if not prefix:
+            # PseudoSingle generally does not get a prefix. Fix that here,
+            # or 'notepad_setpoint' and 'notepad_readback' will have no
+            # prefix.
+            prefix = parent.prefix
+
+        super().__init__(prefix=prefix, parent=parent, **kwargs)
+
+        # TODO: update the notepad PVs in this class
```

Comment:
It's not clear to me that there is a great way to implement this here, since it is the parent PseudoPositioner class that actually controls all the set/get logic

Reply:
Think we need a subclass of PseudoPositioner to handle these?

Reply:
I'll take a look and see what's necessary for this - 1st pass was "see if they like the notepad idea and show how it could be integrated"; 2nd pass will be "actually make it work" 😄

Reply:
Resolved in the reasonably minimal subclass of 02d4e83